psp/exp_configs/island.py

```diff
@@ -0,0 +1,58 @@
+import functools
+
+from psp.data.data_sources.pv import NetcdfPvDataSource, PvDataSource
+from psp.dataset import Splits, split_train_test
+from psp.exp_configs.base import ExpConfigBase
+from psp.models.base import PvSiteModel, PvSiteModelConfig
+from psp.models.recent_history import RecentHistoryModel
+from psp.models.regressors.decision_trees import SklearnRegressor
+from psp.typings import Horizons
+
+PV_DATA_PATH = "data/island/data_15min_FIN.nc"
+NWP_DATA_PATH = "data/island/nwp-v1"
```

Comment:
now "island_nwp_5y"

Reply:
Cool! I'll remove it actually as it is not used in here. You can add it in the PR adding NWP.